qlty-cli/src/commands/init.rs

```diff
@@ -41,13 +41,21 @@ pub struct Init {
     #[arg(long, value_parser = SourceSpec::new)]
     pub source: Option<SourceSpec>,
 
+    /// Warning: this option has been deprecated!
```

Comment:
We can actually hide options from the auto-generated Clap output, which I think is better than a warning 

```rust
#[arg(hide = true, long)]
pub with_prefixes: bool,
```